translator/trace/zipkin/zipkinv1_to_protospan.go

```diff
@@ -250,20 +250,24 @@ func zipkinV1BinAnnotationsToOCAttributes(binAnnotations []*binaryAnnotation) (a
 	return attributes, status, fallbackServiceName
 }
 
-func parseAnnotationValue(value string) *tracepb.AttributeValue {
+func parseAnnotationValue(value string, parseStringTags bool) *tracepb.AttributeValue {
 	pbAttrib := &tracepb.AttributeValue{}
 
-	switch tracetranslator.DetermineValueType(value, false) {
-	case pdata.AttributeValueINT:
-		iValue, _ := strconv.ParseInt(value, 10, 64)
-		pbAttrib.Value = &tracepb.AttributeValue_IntValue{IntValue: iValue}
-	case pdata.AttributeValueDOUBLE:
-		fValue, _ := strconv.ParseFloat(value, 64)
-		pbAttrib.Value = &tracepb.AttributeValue_DoubleValue{DoubleValue: fValue}
-	case pdata.AttributeValueBOOL:
-		bValue, _ := strconv.ParseBool(value)
-		pbAttrib.Value = &tracepb.AttributeValue_BoolValue{BoolValue: bValue}
-	default:
+	if parseStringTags {
```

Comment:
@owais this changes Zipkinv1 receiver behavior. Any concerns with this breaking something that we care about?